pkgs/build-support/fetchurl/mirrors.nix

```diff
@@ -70,13 +70,14 @@ rec {
     ftp://ftp.funet.fi/pub/mirrors/ftp.kernel.org/pub/
   ];
 
-  # Mirrors of ftp://ftp.kde.org/pub/kde/.
+  # Mirrors from https://download.kde.org/extra/download-mirrors.html
   kde = [
-    "http://download.kde.org/download.php?url="
-    http://ftp.gwdg.de/pub/x11/kde/
-    ftp://ftp.heanet.ie/mirrors/ftp.kde.org/
-    ftp://ftp.kde.org/pub/kde/
-    ftp://ftp.funet.fi/pub/mirrors/ftp.kde.org/pub/kde/
```

Comment:
Swapped to HTTP, which is likely faster, and more likely to make it through firewalls.